modcc/perfvisitor.hpp

```diff
@@ -41,6 +41,7 @@ static std::ostream& operator << (std::ostream& os, FlopAccumulator const& f) {
 
 class FlopVisitor : public Visitor {
 public:
+    using Visitor::visit;
```

Comment:
again same thing: is this a bug in nvc++, or what do I overlook?

Reply:
See above.